py/torch_tensorrt/_compile.py

```diff
@@ -36,6 +37,7 @@ class _IRType(Enum):
     fx = 1
     dynamo = 2
     torch_compile = 3
+    exported_program = 4
```

Comment:
Unrelated to this change, but the docstring on this `Enum` is not relevant to the content